.travis.yml

```diff
@@ -1,38 +1,44 @@
 language: python
 
-python:
-  - "2.7"
+sudo: false
 
-env:
-  - CONDA="python=2.7"
+matrix:
+  fast_finish: true
+  include:
+  - python: 2.7
+    env: TEST_TARGET=default
+  - python: 3.5
+    env: TEST_TARGET=default
+  - python: 3.6
+    env: TEST_TARGET=default
+  - python: 3.6
+    env: TEST_TARGET=coding_standards
+  allow_failures:
+  - python: 3.5
+    env: TEST_TARGET=default
+  - python: 3.6
+    env: TEST_TARGET=default
+  - python: 3.6
+    env: TEST_TARGET=coding_standards
```

Comment:
I created a full build matrix. If the code is not Python 3 compliant yet I can move the Python 3 items to the `allow_failures` section and work on them in another PR.